### ISSUE TITLE
Bug: New update broke 'Send later' feature

### ISSUE THREAD

**Issue by sanjeevverma1** [nylas/N1#2106]
I think the new update (0.4.32) broke the 'send later' feature in N1...now when I schedule an email to be sent at a later time, the scheduled message just appears as a draft and no longer displays the time I scheduled it to be sent. 

**Specs**
Nylas N1 Version: 0.4.32-1de17e7 (0.4.32-1de17e7)
OSX 10.11.5

**Comment by nielsvr** [comment]
Got the same issue.
Versie 0.4.32-1de17e7 (0.4.32-1de17e7)
OSX 10.11.4

**Comment by bengotow** [comment]
Hey folks—thanks for reporting this! We're looking in to this issue this morning. Does it matter whether you "Send Later" from a popout composer or the main window?

**Comment by nielsvr** [comment]
@bengotow there is no difference. Both methods will put a post in draft, but no scheduled time/date next to it (and it's not getting send either I believe).

**Comment by bengotow** [comment]
This has been fixed in 6d2755329b9b279a4871e4e1b0ac706107533300 and we'll ship it this week in a patch. Thanks for reporting!

**Comment by enoversum** [comment]
I experienced this yesterday as well, though a bit differently, @bengotow . 
I wrote an e-mail late last night, chose to send it 'tomorrow morning'. After clicking the option, nothing happened, and the e-mail was sent right away when I ... sent it. This was in a reply within a conversation. I can reproduce this today on another computer.
If I create a new e-mail, the feature seems to work, i.e. e-mail is being closed and put to drafts right away (though with no indication as that it will be sent later).
